### ISSUE TITLE
The tippy content text is blurry.

### ISSUE THREAD

**Issue by rromikas** [tvkhoa/react-tippy#141]
The tippy content text is  @blurry. Is a fix for it?

**Comment by haneenmahd** [comment]
Yeah, the content inside it is blurry when it is small.but when I resized the page the blur was no more to be seen

**Comment by kaelansmith** [comment]
This solved it for me: 

The blurry text is a known issue with Popper.js, and this library provides the 'popperOptions' prop, but unfortunately doesn't document it at all; so after some trial and error I found the correct syntax for fixing the blur:

`popperOptions={{
  modifiers: {
     computeStyle: {
        gpuAcceleration: false
     }
  }
}}`

Caveat: I don't know if setting this to false has any adverse effects.
